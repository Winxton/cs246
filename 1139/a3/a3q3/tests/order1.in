! a
+ a b + b c
+ a d
p
