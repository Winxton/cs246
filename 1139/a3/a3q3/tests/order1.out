a is parent of b,d
b is parent of c
c has no children
d has no children
